<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Savvy" failures="1" tests="2" name="Test" time="26.864" errors="0" timestamp="2025-01-27T13:57:23 IST">
  <testcase classname="testCases.TC_002_LoginTest" name="verify_login" time="3.36"/>
  <testcase classname="testCases.TC_002_LoginTest" name="verify_login2" time="11.552">
    <failure type="java.lang.AssertionError" message="Login failed expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Login failed expected [true] but found [false]
at testCases.TC_002_LoginTest.verify_login2(TC_002_LoginTest.java:77)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- verify_login2 -->
</testsuite> <!-- Test -->
